pkg/library/status/check.go

```diff
@@ -181,3 +181,12 @@ func checkPodConditions(pod *corev1.Pod) (msg string) {
 	}
 	return ""
 }
+
+func getEventCount(event corev1.Event) int32 {
```

Comment:
Could you include a quick doc explaining the reasoning for this function (i.e. how we came to the end of this confusing road on event counts :P )?

Reply:
would be also nice to add a link to the OpenShift issue

Reply:
+1 will do :)